membership/src/main/kotlin/net/corda/v5/membership/MemberInfo.kt

```diff
@@ -36,15 +37,25 @@ interface MemberInfo {
      */
     val sessionInitiationKey: PublicKey
 
-    /** List of current and previous (rotated) ledger keys, which member can still use to sign unspent transactions on ledger. */
+    /**
+     * List of current and previous (rotated) ledger keys, which member can still use to sign unspent
+     * transactions on ledger.
+     * Key at index 0 is always the currently active ledger key.
```

Comment:
Is it good to describe like that? As far as I remember all keys are "active" - can be used to sign. Would it be better to describe it as the latest? or To describe that the order of the keys would be with the latest generated at the top?

Reply:
Fair point. I'll change wording to `latest`

Reply:
Updated